OloEditor/src/Panels/StreamingPanel.cpp

```
@@ -0,0 +1,300 @@
+            return;
+        }
+
+        DrawSettingsSection();
+        DrawExportSection();
+        DrawRegionsSection();
+        DrawDebugSection();
+
+        ImGui::End();
+    }
+
+    void StreamingPanel::DrawSettingsSection()
```

Clang (14.0.6)

[warning] method 'DrawSettingsSection' can be made static

(readability-convert-member-functions-to-static)